crates/dap/src/client.rs

```diff
@@ -534,14 +534,14 @@ impl DebugAdapterClient {
 
     pub async fn set_breakpoints(
         &self,
-        path: PathBuf,
-        breakpoints: Option<Vec<SourceBreakpoint>>,
```

Comment:
I'm open to changing it back, but I don't think that edge case is an issue. Whenever a breakpoint is toggled, updated breakpoint information is sent to active adapters. The only other case where breakpoints are sent to an adapter is during configuration which isn't affected by this change.

If we need to remove all breakpoints from a file we can send an empty Vec instead of None.